### ISSUE TITLE
Build wheels for Python 3.9

### ISSUE THREAD

**Issue by Gowee** [cher-nov/cryptg#5]
Hi!

`cibuildwheel` is ready for Python 3.9 (https://github.com/joerick/cibuildwheel/pull/440 ). Also, I have tested `cryptg` on Python 3.9 where it builds well. It would be great if `cryptg` can provide with prebuilt wheels for Python 3.9.

Thanks.

**Comment by Lonami** [comment]
Update on this, Linux and macOS builds are now available (https://pypi.org/project/cryptg/0.2.post2/#files), but we're having trouble with the Windows build (https://ci.appveyor.com/project/cher-nov/cryptg/builds/36088966):

<details>

```
Build started
git clone -q --branch=master https://github.com/cher-nov/cryptg.git C:\projects\cryptg
git checkout -qf 6bc358b3984f75a3fbf3c90116b4a04ab38cae1b
Running Install scripts
git submodule update --init --recursive
Submodule 'share/tiny-AES-c' (https://github.com/cher-nov/tiny-AES-c.git) registered for path 'share/tiny-AES-c'
Cloning into 'C:/projects/cryptg/share/tiny-AES-c'...
Submodule path 'share/tiny-AES-c': checked out '58cbfcbfec24e74780bff35d6c9aa87d17700805'
pip install twine cibuildwheel
DEPRECATION: Python 2.7 will reach the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 won't be maintained after that date. A future version of pip will drop support for Python 2.7. More details about Python 2 support in pip, can be found at https://pip.pypa.io/en/latest/development/release-process/#python-2-support
Collecting twine
  Downloading https://files.pythonhosted.org/packages/23/0e/9e833399b84e9232a7f86a8f4351d6687dbab0b03a2a6e352d431463fa68/twine-1.15.0-py2.py3-none-any.whl
Collecting cibuildwheel
  Downloading https://files.pythonhosted.org/packages/65/09/a7885588a249020d0101b00a564e2ab738a135c4f05fd5c4f2c77024f73d/cibuildwheel-1.1.0-py2.py3-none-any.whl
Collecting requests-toolbelt!=0.9.0,>=0.8.0
  Downloading https://files.pythonhosted.org/packages/60/ef/7681134338fc097acef8d9b2f8abe0458e4d87559c689a8c306d0957ece5/requests_toolbelt-0.9.1-py2.py3-none-any.whl (54kB)
Collecting readme-renderer>=21.0
  Downloading https://files.pythonhosted.org/packages/54/18/0adfcac99c08a6d45947e5da4e1388a7bb9957502a2b4ad12176fed6aeaf/readme_renderer-28.0-py2.py3-none-any.whl
Collecting pkginfo>=1.4.2
  Downloading https://files.pythonhosted.org/packages/4f/3c/535287349af1b117e082f8e77feca52fbe2fdf61ef1e6da6bcc2a72a3a79/pkginfo-1.6.1-py2.py3-none-any.whl
Collecting requests!=2.15,!=2.16,>=2.5.0
  Downloading https://files.pythonhosted.org/packages/45/1e/0c169c6a5381e241ba7404532c16a21d86ab872c9bed8bdcd4c423954103/requests-2.24.0-py2.py3-none-any.whl (61kB)
Requirement already satisfied: setuptools>=0.7.0 in c:\python27\lib\site-packages (from twine) (41.2.0)
Collecting tqdm>=4.14
  Downloading https://files.pythonhosted.org/packages/93/3a/96b3dc293aa72443cf9627444c3c221a7ba34bb622e4d8bf1b5d4f2d9d08/tqdm-4.51.0-py2.py3-none-any.whl (70kB)
Collecting bashlex!=0.13
  Downloading https://files.pythonhosted.org/packages/c1/0d/ff84a5cf5822b12d77ab08ab91b2b00550a7ca742aa9d31381b7c88deb0f/bashlex-0.15.tar.gz (60kB)
Collecting six
  Downloading https://files.pythonhosted.org/packages/ee/ff/48bde5c0f013094d729fe4b0316ba2a24774b3ff1c52d924a8a4cb04078a/six-1.15.0-py2.py3-none-any.whl
Collecting bleach>=2.1.0
  Downloading https://files.pythonhosted.org/packages/03/c8/b7ed0dfea5cb287907bd22c5ff7c3ed0a65b346f2a4cf916eb9e83be66b3/bleach-3.2.1-py2.py3-none-any.whl (145kB)
Collecting Pygments>=2.5.1
  Downloading https://files.pythonhosted.org/packages/be/39/32da3184734730c0e4d3fa3b2b5872104668ad6dc1b5a73d8e477e5fe967/Pygments-2.5.2-py2.py3-none-any.whl (896kB)
Collecting docutils>=0.13.1
  Downloading https://files.pythonhosted.org/packages/81/44/8a15e45ffa96e6cf82956dd8d7af9e666357e16b0d93b253903475ee947f/docutils-0.16-py2.py3-none-any.whl (548kB)
Collecting certifi>=2017.4.17
  Downloading https://files.pythonhosted.org/packages/5e/c4/6c4fe722df5343c33226f0b4e0bb042e4dc13483228b4718baf286f86d87/certifi-2020.6.20-py2.py3-none-any.whl (156kB)
Collecting chardet<4,>=3.0.2
  Downloading https://files.pythonhosted.org/packages/bc/a9/01ffebfb562e4274b6487b4bb1ddec7ca55ec7510b22e4c51f14098443b8/chardet-3.0.4-py2.py3-none-any.whl (133kB)
Collecting idna<3,>=2.5
  Downloading https://files.pythonhosted.org/packages/a2/38/928ddce2273eaa564f6f50de919327bf3a00f091b5baba8dfa9460f3a8a8/idna-2.10-py2.py3-none-any.whl (58kB)
Collecting urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1
  Downloading https://files.pythonhosted.org/packages/56/aa/4ef5aa67a9a62505db124a5cb5262332d1d4153462eb8fd89c9fa41e5d92/urllib3-1.25.11-py2.py3-none-any.whl (127kB)
Collecting enum34
  Downloading https://files.pythonhosted.org/packages/6f/2c/a9386903ece2ea85e9807e0e062174dc26fdce8b05f216d00491be29fad5/enum34-1.1.10-py2-none-any.whl
Collecting webencodings
  Downloading https://files.pythonhosted.org/packages/f4/24/2a3e3df732393fed8b3ebf2ec078f05546de641fe1b667ee316ec1dcf3b7/webencodings-0.5.1-py2.py3-none-any.whl
Collecting packaging
  Downloading https://files.pythonhosted.org/packages/46/19/c5ab91b1b05cfe63cccd5cfc971db9214c6dd6ced54e33c30d5af1d2bc43/packaging-20.4-py2.py3-none-any.whl
Collecting pyparsing>=2.0.2
  Downloading https://files.pythonhosted.org/packages/8a/bb/488841f56197b13700afd5658fc279a2025a39e22449b7cf29864669b15d/pyparsing-2.4.7-py2.py3-none-any.whl (67kB)
Installing collected packages: certifi, chardet, idna, urllib3, requests, requests-toolbelt, six, webencodings, pyparsing, packaging, bleach, Pygments, docutils, readme-renderer, pkginfo, tqdm, twine, enum34, bashlex, cibuildwheel
    Running setup.py install for bashlex: started
    Running setup.py install for bashlex: finished with status 'done'
Successfully installed Pygments-2.5.2 bashlex-0.15 bleach-3.2.1 certifi-2020.6.20 chardet-3.0.4 cibuildwheel-1.1.0 docutils-0.16 enum34-1.1.10 idna-2.10 packaging-20.4 pkginfo-1.6.1 pyparsing-2.4.7 readme-renderer-28.0 requests-2.24.0 requests-toolbelt-0.9.1 six-1.15.0 tqdm-4.51.0 twine-1.15.0 urllib3-1.25.11 webencodings-0.5.1
WARNING: You are using pip version 19.3.1; however, version 20.2.4 is available.
You should consider upgrading via the 'python -m pip install --upgrade pip' command.
IF DEFINED CI_TEST_LAUNCH SET "PYPI_SETUP_VERSION_SUFFIX=dev999%APPVEYOR_BUILD_ID%";
cibuildwheel --output-dir "%BUILD_OUTPUT_PATH%"
     _ _       _ _   _       _           _
 ___|_| |_ _ _|_| |_| |_ _ _| |_ ___ ___| |
|  _| | . | | | | | . | | | |   | -_| -_| |
|___|_|___|___|_|_|___|_____|_|_|___|___|_|
cibuildwheel version 1.1.0
Build options:
  platform: 'windows'
  before_build: None
  build_selector: BuildSelector('*' - '?p27-*')
  build_verbosity: 3
  environment: ParsedEnvironment([])
  output_dir: 'wheelhouse'
  project_dir: '.'
  repair_command: ''
  test_command: None
  test_extras: ''
  test_requires: []
Here we go!
+ Download https://dist.nuget.org/win-x86-commandline/latest/nuget.exe to C:\cibw\nuget.exe
Traceback (most recent call last):
  File "c:\python27\lib\runpy.py", line 174, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "c:\python27\lib\runpy.py", line 72, in _run_code
    exec code in run_globals
  File "C:\Python27\Scripts\cibuildwheel.exe\__main__.py", line 7, in <module>
  File "c:\python27\lib\site-packages\cibuildwheel\__main__.py", line 180, in main
    cibuildwheel.windows.build(**build_options)
  File "c:\python27\lib\site-packages\cibuildwheel\windows.py", line 96, in build
    download('https://dist.nuget.org/win-x86-commandline/latest/nuget.exe', nuget)
  File "c:\python27\lib\site-packages\cibuildwheel\windows.py", line 70, in download
    response = urlopen(url)
  File "c:\python27\lib\urllib2.py", line 154, in urlopen
    return opener.open(url, data, timeout)
  File "c:\python27\lib\urllib2.py", line 429, in open
    response = self._open(req, data)
  File "c:\python27\lib\urllib2.py", line 447, in _open
    '_open', req)
  File "c:\python27\lib\urllib2.py", line 407, in _call_chain
    result = func(*args)
  File "c:\python27\lib\urllib2.py", line 1241, in https_open
    context=self._context)
  File "c:\python27\lib\urllib2.py", line 1198, in do_open
    raise URLError(err)
urllib2.URLError: <urlopen error [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:727)>
Command exited with code 1
```

</details>

**Comment by cher-nov** [comment]
This is actually already fixed in [cibuildwheel 1.6.4](https://github.com/joerick/cibuildwheel/releases/tag/v1.6.4) after being noted in https://github.com/joerick/cibuildwheel/issues/452, https://github.com/joerick/cibuildwheel/issues/456 and maybe partially https://github.com/joerick/cibuildwheel/issues/443. But for some reason the AppVeyor still downloads the pretty old version of it (1.1.0), as it can be seen from the build log. Trying to investigate that now.

**Comment by cher-nov** [comment]
https://ci.appveyor.com/project/cher-nov/cryptg/builds/36091162
https://pypi.org/project/cryptg/0.2.post2/#files

Seems to be done now. Please test.
